sdk/ledgerjs-hw-app-sui/README.md

```diff
@@ -0,0 +1,129 @@
+[Github](https://github.com/LedgerHQ/ledgerjs/),
+[Ledger OP3N Discord](https://discord.gg/hTy7ZXvR7y)
```

Comment:
Are these links still correct?

Reply:
Yes, but I made a few updates here. Just noting that we'll improve the `README.md` at some point as well 😄